dist/about.html

```diff
@@ -28,69 +28,72 @@
       <span class="menu-btn__burger"></span>
```

Comment:
- [ ] In desktop view the about page has a white background according to the design. Kindly address this issue.

Given Design
![image](https://user-images.githubusercontent.com/81932478/169595161-d02fb76a-7a46-4981-9031-cfa27efab432.png)

Your Design
![image](https://user-images.githubusercontent.com/81932478/169595225-be5e0fd4-a26b-4581-ab6c-0628bfb7086e.png)